pages/linux/procs.md

```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
+> More information: <https://github.com/dalance/procs>.
+
+- List all processes with PID, user, CPU usage, memory usage and the command which started them:
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:
+
+`procs --sort{{d}}|--sort{{a}} cpu`
+
+- Show information about processes with PID `41` or (command or user) containing (`zsh` `or` `firefox`):
+
+`procs --or {{zsh}} {{41}} {{firefox}}`
+
+- Show information about processes with PID `41` and (command or user) containing `zsh`:
```

Comment:
```suggestion
- Show information about processes with either a PID, command or, user containing `zsh`:
```

Reply:
`either` can only be used with `or` and we have `and` here, so it should be `both`. Also, we can't use the Oxford comma here, since it's not part of the enumeration.
```suggestion
- Show information about processes with both PID `41` and a command or user containing `zsh`:
```